internal/pkg/levee/propagation/propagation.go

```diff
@@ -145,6 +145,11 @@ func (prop *Propagation) visit(n ssa.Node, maxInstrReached map[*ssa.BasicBlock]i
 		}
 
 	case *ssa.Call:
+		// The builtin delete(m map[Type]Type1, key Type) func does not propagate source.
```

Comment:
Nit: ... does not propagate *taint*. (If this is just a typo, ignore the following: a `source` is a value that is itself sensitive, it is the starting point of a `taint` propagation analysis; `taint` propagates from value to value, starting from a `source`)

Reply:
Updated the doc.